ui/admin/app/components/form/alias/index.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+
+export default class FormAliasComponent extends Component {
+  //actions
+  /**
+   * Handles input changes
+   */
+  @action
+  handleInputChange({ target: { value } }) {
```

Comment:
Can we use a more specific name here? i.e: `handleHostIdChange` or something like that? `handleInputChange` seems to generalistic? can be any field of the form.